Documentation/devicetree/bindings/sound/everest,es8326.yaml

```diff
@@ -0,0 +1,110 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/sound/everest,es8326.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Everest ES8326 audio CODEC
+
+maintainers:
+  - Daniel Drake <drake@endlessm.com>

```

Comment:
really? Daniel was not part of this development effort, was he?

Reply:
removed @plbossart